lib/apiservers/engine/backends/image.go

```diff
@@ -147,8 +155,22 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
-	client := PortLayerClient()
-	ImageCache().Update(client)
+	imageJSON, err := ioutil.ReadFile(imageFile)
```

Comment:
@dougm So there's other stuff coming through stdout (progress, digest, etc). Would this provide any way of ignoring the stuff I don't want?